Makefile

```diff
@@ -1,21 +1,40 @@
 all:
-	npm run build
+	NODE_ENV=$(NODE_ENV) npm run build
 
 clean:
 	rm -rf dist/
 	rm -rf _install
 
 install: all
-	mkdir -p /usr/share/cockpit
-	cp -r dist/ /usr/share/cockpit/subscription-manager
+	mkdir -p /usr/share/cockpit/subscription-manager
+	cp -r dist/* /usr/share/cockpit/subscription-manager
 
-srpm: clean all
+dist-gzip: NODE_ENV=production
+dist-gzip: clean all
```

Comment:
No, this calls `clean` and `all` with the environment variable set to production. See [target-specific variable values](https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html)